atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraService.java

```diff
@@ -102,13 +104,30 @@ public class CassandraService implements AutoCloseable {
     private final Supplier<Map<String, String>> hostnameByIpSupplier;
 
     private final Random random = new Random();
+    private final CassandraClientInstrumentation cassandraClientInstrumentation;
 
-    public CassandraService(
+    public static CassandraService create(
             MetricsManager metricsManager,
             CassandraKeyValueServiceConfig config,
             Refreshable<CassandraKeyValueServiceRuntimeConfig> runtimeConfig,
             Blacklist blacklist,
             CassandraClientPoolMetrics poolMetrics) {
+        return new CassandraService(
+                metricsManager,
+                config,
+                runtimeConfig,
+                blacklist,
+                poolMetrics,
+                new UnfilteredCassandraClientInstrumentation(metricsManager.getTaggedRegistry()));
```

Comment:
Any reason you chose to take in poolMetrics but build the instrumentation here?

Reply:
I didn't think about poolMetrics here as I don't use it. I was aiming for a minimal diff & to wire things correctly.

Also, this place in the code will change slightly as there will be a `createForTests` variant. That one uses the unfiltered one, the other uses the filtered one. You could have them passed but I don't really mind, so I went for the smallest diff!

When you look at the last PR, let me know how you want me to change this if at all!